app/design/adminhtml/Magento/backend/web/css/styles-old.less

```diff
@@ -984,22 +984,10 @@
         width: 20%;
         padding-top: 8px;
         padding-right: 30px;
-
-        &.admin__field-label {
-            padding-top: 0;
-            padding-right: 30px;
-            width: 20%;
-            margin-left: 0;
-        }
-    }
-
-    .form-inline .label ~ .control {
-        margin: 0;
-        width: 60%;
     }
 
     .form-inline .no-label .control {
-        margin-left: 20%;
+        margin-left: 26.7%;
```

Comment:
Why so strange size? Looks like you try to adapt it to some view size, but it will not work on another screen sizes.